### PR TITLE
Add hold_key and release_key to support alt+tab switcher

### PR DESCRIPTION
**Pull request by mitiao**
follow send_key to implement the hold_key and release_key to support alt+tab 
this request came from https://progress.opensuse.org/issues/11326
and the test result:
http://147.2.207.208/tests/1038

### CODE REVIEW

File: backend/driver.pm [hunk 1]
```diff
@@ -144,6 +144,16 @@ sub send_key {
     return $self->_send_json({cmd => 'send_key', arguments => {key => $key}});
 }
 
+sub hold_key {
```
Comment (coolo) [inline backend/driver.pm]: you don't need these if you call the backend function as hold_key({key => $key}). The send_key function is historic

File: consoles/VNC.pm [hunk 2]
```diff
@@ -686,7 +686,8 @@ sub init_ikvm_keymap {
 
 
 sub map_and_send_key {
-    my ($self, $keys) = @_;
+    my ($self, $keys, $down_flag) = @_;
+    $down_flag //= '';
```
Comment (coolo) [inline consoles/VNC.pm]: Please don't. Use defined to check for undefined values

File: backend/driver.pm (hunk 1 above)
Comment (mitiao) [inline backend/driver.pm]: Ah, I see.
Thanks.

File: consoles/VNC.pm (hunk 2 above)
Comment (mitiao) [inline consoles/VNC.pm]: Fixed

**Pull request merged**